js/app/modules/controller/course.js

```diff
@@ -0,0 +1,63 @@
+// Copyright 2015 Endless Mobile, Inc.
+
+/* exported Course */
+
+const GObject = imports.gi.GObject;
+
+const CourseHistoryStore = imports.app.courseHistoryStore;
+const Engine = imports.search.engine;
+const Controller = imports.app.interfaces.controller;
+const HistoryStore = imports.app.historyStore;
+const Module = imports.app.interfaces.module;
+const Pages = imports.app.pages;
+const ReadingHistoryModel = imports.app.readingHistoryModel;
+const SetMap = imports.app.setMap;
+const QueryObject = imports.search.queryObject;
+
+/**
+ * Class: Course
+ * Controller that presents content as a course to be consumed.
+ *
+ * Implements:
+ *    <Module>, <Controller>
+ */
+const Course = new Module.Class({
```

Comment:
With Course and Buffet, I saw we unnaturally force all controller names to match a dining theme. We can start with Mesh -> Strainer.